src/map.h

```diff
@@ -2055,6 +2055,9 @@ class map
         std::vector<item *> get_active_items_in_radius( const tripoint &center, int radius,
                 special_item_type type ) const;
 
+        /** returns positions of furnitures with matching flag in the overmap terrain*/
+        std::list<tripoint> find_furnitures_with_flag_in_omt( const tripoint &p, const std::string &flag );
```

Comment:
```suggestion
        std::vector<tripoint> find_furnitures_with_flag_in_omt( const tripoint &p, const std::string &flag );
```
would be better to use vector in most cases since lists are often less performant due to cache locality